gammapy/maps/wcsnd.py

```diff
@@ -528,6 +528,65 @@ def plot(self, ax=None, fig=None, add_cbar=False, stretch='linear', **kwargs):
         ax.autoscale(enable=False)
         return fig, ax, cbar
 
+
+    def plot_interactive(self, ax=None, fig=None, **kwargs):
```

Comment:
Please add at least a one line summary docstring.
Or even a full docstring with some short explanations what this is.

Reply:
Done.